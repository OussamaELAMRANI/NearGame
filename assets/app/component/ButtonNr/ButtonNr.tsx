import { FC } from 'react'

interface ButtonProps {

}

const ButtonNr: FC<ButtonProps> = ({ children, ...props }) => {
  return (<button {...props}> {children} </button>)
}

export default ButtonNr
