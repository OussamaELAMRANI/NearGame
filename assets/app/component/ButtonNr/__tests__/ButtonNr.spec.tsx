import { cleanup } from '@testing-library/react'
import { create } from 'react-test-renderer'

import ButtonNr from '../index'

describe('ButtonNr.tsx', () => {
  afterEach(cleanup)

  it('should render correctly !', () => {
    const elem = create(<ButtonNr>my button</ButtonNr>).toJSON()
    expect(elem).toMatchSnapshot()
  })
})
