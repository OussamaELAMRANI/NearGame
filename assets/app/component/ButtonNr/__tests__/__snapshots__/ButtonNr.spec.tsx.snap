// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonNr.tsx should render correctly ! 1`] = `
<button>
   
  my button
   
</button>
`;
